src/ucp/api/ucp.h

```diff
@@ -1079,6 +1101,44 @@ struct ucp_tag_recv_info {
 };
 
 
+typedef struct {
+    /**
+     * Mask of valid fields in this structure and operation flags, using
+     * bits from @ref ucp_op_attr_t. Fields not specified in this mask will be
+     * ignored. Provides ABI compatibility with respect to adding new fields.
+     */
+    uint64_t       attr_mask;
```

Comment:
maybe rename it to "op_attr_mask" to be consistent with "ucp_op_attr_t" type?

Reply:
ok, renamed